### DISCUSSION TITLE
Hide folder from the sidebar?

### DISCUSSION THREAD

**Discussion by softshape** [facebook/docusaurus#5768] [Q&A]
When the sidebar is autogenerated, is there any way to hide a folder or document from it? I need to separate some part of docs out of the sidebar tree, yet keep it accessible by direct URL.

**Comment by Josh-Cena** [comment]
https://github.com/facebook/docusaurus/issues/5689

Still at the RFC stage. I plan to think more about the API after we have "category index pages" (which should be next week hopefully), since there are more use-cases we need to support

**Comment by softshape** [comment]
May I suggest this syntax?

```
---
sidebar_position: hide
---
```

- this would allow to show and hide an exact article; if all the docs in the folder are hidden, the whole folder is too.

Besides of that, right now I can only imagine manual sidebar creation, but if you can suggest any workaround that would be great.

**Reply by Josh-Cena** [reply]
That would probably be part of this: https://github.com/facebook/docusaurus/issues/5686

So you can set `sidebar: null` or something like that to generate the link while preventing the sidebar from being displayed.

There has been quite some complications around the sidebar logic, both to the users and to the implementers. I think we need to formalize all the concepts / explanations
